### PR TITLE
TSERV-248 : Renaming artifacts to swagger-assert4j-...

### PR DESCRIPTION
**Pull request by prakashmrec**

### CODE REVIEW

File: modules/swagger/pom.xml [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-        <artifactId>assert4j</artifactId>
+        <artifactId>swagger-assert4j</artifactId>
         <groupId>io.swagger.assert</groupId>
         <version>1.0.0-SNAPSHOT</version>
         <relativePath>../..</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
     <artifactId>assert4j-swagger</artifactId>
-    <name>assert4j-swagger</name>
+    <name>swagger-assert4j-swagger</name>
```
Comment (prakashmrec) [inline modules/swagger/pom.xml]: This is weird name, we should probably rename it to something else. @olensmar ?

**Comment by prakashmrec** [comment]
We can rename the repo once this PR is in.

File: modules/swagger/pom.xml (hunk 1 above)
Comment (olensmar) [inline modules/swagger/pom.xml]: hm... swagger-assert4j-oas perhaps?

Comment (prakashmrec) [inline modules/swagger/pom.xml]: Sounds good, I will fix.

**Pull request merged**